docs/DeveloperGuide.adoc

```diff
@@ -729,9 +729,83 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |set certain contacts as favourites |access to their information easily
+
+|`* * *` |user |select multiple contacts to delete |clean up address book efficiently
+
+|`* * *` |user |save more than one contact number per contact in my address book |account for multiple phone numbers
+
+|`* * *` |organised user |group contacts manually or automatically |access the information I want easily
+
+|`* * *` |user |see my important events on a calendar view |overview of my schedule for the month
+
+|`* * *` |user |share my contact details with my friends |friends do not need to manually add it in
+
+|`* * *` |user |sort the information on the application |view the information in a ordered manner
+
+|`* * *` |user |add information to the application in a intuitive way |not frustrating
+
+|`* * *` |user |sync my account with other platforms |save the trouble of entering my personal details
+
+|`* * *` |user |access my data over multiple devices |do not have key in my data repeatedly
+
+|`* * *` |user |have my account logged in every time |do not have to remember one additional username and password
+
+|`* * *` |user |set reminders |do not forget what I need to do
+
+|`* * *` |user |export the list of people in my address book |store it on the computer
+
+|`* * *` |user |set events and send the event invites to any person in my contact list |calendars can sync the events
+
+|`* * *` |user |attach personal notes to people in my contact list |take notes about the person for my own reference
+
+|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else
+seeing them by accident
+
+|`* *` |user |send mass messages to a group of contacts |plan for meetings, conferences or gatherings
+
+|`* *` |user |see my groupings in a more visual display |see who are also in the group
```

Comment:
I have changed this to two user stories.
1. As a lazy user, I can drag and drop contacts in or out of different groups so that I do not have to key in complex command lines.
2.As a user, I can view groups of contact in a more visual display so that I can refer to a contact in a group easily.

Reply:
Done